test/e2e/load.go

```diff
@@ -94,17 +81,45 @@ var _ = Describe("Load capacity", func() {
 
 	type Load struct {
 		podsPerNode int
+		image       string
+		command     []string
+		skip        bool
+		deleteNs    bool
 	}
 
 	loadTests := []Load{
-		{podsPerNode: 30},
+		// The container will consume 1 cpu and 512mb of memory.
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
+		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:1.1", skip: true, deleteNs: true},
 	}
 
 	for _, testArg := range loadTests {
-		name := fmt.Sprintf("[Skipped] should be able to handle %v pods per node", testArg.podsPerNode)
+		name := fmt.Sprintf("should be able to handle %v pods per node", testArg.podsPerNode)
+		if testArg.podsPerNode == 30 {
```

Comment:
We don't want to introduce another performance test to be able to run Performance suite every hour. That's why we also removed Density test with 3 pods per node from the suite. However we may want to consider running this test if you think it will be useful. @wojtek-t WDYT?